examples/contracts/football_prediction_market.py

```diff
@@ -75,11 +76,13 @@ def get_match_result() -> str:
 your output must be only JSON without any formatting prefix or suffix.
 This result should be perfectly parsable by a JSON parser without errors.
             """
-            result = gl.exec_prompt(task).replace("```json", "").replace("```", "")
+            result = (
+                gl.nondet.exec_prompt(task).replace("```json", "").replace("```", "")
+            )
             print(result)
-            return json.dumps(json.loads(result), sort_keys=True)
+            return json.loads(result)
```

Comment:
@cristiam86 here is the fix that avoids dumps... I remember you wanted to introduce it